lib/fastboot-info.js

```diff
@@ -30,6 +31,10 @@ FastBootInfo.prototype.extractCookies = function(request) {
   return {};
 };
 
+FastBootInfo.prototype.extractHost = function(request) {
+  return `${request.protocol}://${request.headers.host}`;
```

Comment:
Shouldn't this be `request.hostname` instead of `request.headers.host`?

From the [Express docs](http://expressjs.com/en/api.html#req.hostname):

> *req.hostname*
> Contains the hostname derived from the Host HTTP header.